titus-server-master/src/main/java/com/netflix/titus/master/mesos/kubeapiserver/direct/DirectKubeConfiguration.java

```diff
@@ -64,4 +64,10 @@
      */
     @DefaultValue("false")
     boolean isKubeApiServerIntegratorOldPodCreationEnabled();
+
+    /**
+     * Set to true to enable EBS PV and PVC management.
+     */
+    @DefaultValue("false")
+    boolean isEbsVolumePvEnabled();
```

Comment:
Why do we have this still?
Are you thinking we might "someday" make PVs and such?

Reply:
More of a, "lets actually get this to production" before removing all of the other stuff. Once our path settles, all of that can be removed.